bin/jobsub_cmd

```diff
@@ -113,7 +113,7 @@ def main() -> None:
                 "-factory",
                 "-unmatchable",
             ]
-            or i[0] in "0123456789"
+            or i[:1].isalnum()
```

Comment:
Any reason  this needs to be i[:1] rather than i[0]?  Not saying I object, but just wondering.

Reply:
In my mind f[0] was a character not a string... f[:1] is  a string ... but f[0] 
still is a  string so It doesn't matter...

Reply:
Which is to say, too much C++ lately :-)